### ISSUE TITLE
getAllContacts method not Webpack compatible

### ISSUE THREAD

**Issue by NickIliev** [firescript/nativescript-contacts#46]
@firescript  the method `getAllContacts` is using a `Worker` it must be also made Webpack compatible.

e.g. change [this line ](https://github.com/firescript/nativescript-contacts/blob/master/index.android.js#L106)
```JavaScript
orker = new Worker('./get-all-contacts-worker.js'); // relative for caller script path
```

to
```JavaScript
var worker;
        if (global["TNS_WEBPACK"]) {
            var myWorker = require('nativescript-worker-loader!./get-all-contacts-worker.js.js');
            worker = new myWorker();
        } else {
            worker = new Worker('./get-all-contacts-worker.js'); // relative for caller script path
        }
```

**Comment by erkanarslan** [comment]
@NickIliev After moving my mobile only project to a code sharing project which is based on webpack I cannot use getContact() function. Its success and error promise callbacks aren't called and the code block after the line where I call contacts.getContact() is not executed at all. The function containing all that logic returns immediately.

Do you think that this might be related to the webpack problem you mentioned?

**Comment by jzgoda** [comment]
Resolved by PR #50
